TeamCode/src/main/java/org/firstinspires/ftc/teamcode/bots/FrenzyBot.java

```diff
@@ -104,6 +155,7 @@ public void init(LinearOpMode owner, HardwareMap ahwMap, Telemetry telemetry) th
 
         try {
             this.frenzyConfig = this.botConfig.getFreightFrenzyConfig();
+            this.applyFrenzyConfig();
```

Comment:
It is too early to ask for the config in the constructor. It is processed in the init() method

Reply:
super.init() is already called by this time, so the 'botConfig' object should be populated by this time, isn't it?